### PR TITLE
Added Ascending and Descending sorting to the playlist

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
## Hey, I just made a Pull Request!
Added Ascending and Descending sorting to the playlist

image

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx [hunk 1]
```diff
@@ -40,13 +40,24 @@ describe('PlaylistSortPicker', () => {
...
-    const abcSort = getByText('Alphabetical');
+
+    const abcSort = getByText('A-Z (ascending)');
     expect(abcSort).toBeInTheDocument();
 
     fireEvent.click(abcSort);
     await waitFor(() => {
       expect(updateSort).toHaveBeenLastCalledWith(
-        DefaultSortCompareFunctions[DefaultPlaylistSortTypes.alphabetical],
+        DefaultSortCompareFunctions[DefaultPlaylistSortTypes.ascending],
```
Comment (camilaibs) [inline plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx:50]: I would recommend checking that the items are rendered in the correct order: i.e. that Ambrish is the first item and Ramachandiram the second for ascending order and the opposite for descending order. Just checking that the function was called does not guarantee that its implementation is correct.

**Review by camilaibs — CHANGES_REQUESTED** [review]
Thanks for contributing with us 🙌🏻 .
Just left a small recommendation in the test file

File: plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx (hunk 1 above)
Comment (AmbrishRamachandiran) [reply plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx:90]: @camilaibs modified test case please check

**Review by kuangp — COMMENTED** [review]
I'm wondering if instead of creating separate options for ascending and descending for alphabetical sort specifically, we add some kind of up/down button in the UI so the user can choose asc/desc for any sort type - @camilaibs what do you think? Is this a UI that is seen elsewhere in Backstage? (I've only seen this for table headers)

File: plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx (hunk 1 above)
Comment (camilaibs) [reply plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx:91]: I was actually thinking of something like this:
```tsx
const ascendingItems = getAllByText(/playlist-/);
// expect 1st item is playlist-1
expect(ascendingItems[0]).toHaveTextContent('playlist-1');
// expect 2nd item is playlist-2
expect(ascendingItems[1]).toHaveTextContent('playlist-2');
```

**Comment by camilaibs** [comment]
> I'm wondering if instead of creating separate options for ascending and descending for alphabetical sort specifically, we add some kind of up/down button in the UI so the user can choose asc/desc for any sort type - @camilaibs what do you think? Is this a UI that is seen elsewhere in Backstage? (I've only seen this for table headers)

This sounds like a great idea, thanks for suggesting 💡 !
Would like to try it our @AmbrishRamachandiran  🙂  ?

**Pull request closed without merging**